lib/pure/collections/sequtils.nim

```diff
@@ -704,6 +704,31 @@ template newSeqWith*(len: int, init: untyped): untyped =
     result[i] = init
   result
 
+when not defined(nimscript):
```

Comment:
Why is it a problem for nimscript?

Reply:
@dom96 Error from nim while executing `nim e tests\test_nimscript.nims`
> lib\core\macros.nim(735, 1) Error: cannot generate VM code for macro dumpTree(s: untyped): untyped

Reply:
seems like a bug(?)